src/handlers/swap_handlers.ts

```diff
@@ -403,14 +403,24 @@ const parseGetSwapQuoteRequestParams = (
     });
 
     const affiliateAddress = req.query.affiliateAddress as string;
-    const rfqt: Pick<RfqtRequestOpts, 'intentOnFilling' | 'isIndicative' | 'nativeExclusivelyRFQT'> =
-        takerAddress && apiKey
-            ? {
-                  intentOnFilling: endpoint === 'quote' && req.query.intentOnFilling === 'true',
-                  isIndicative: endpoint === 'price',
-                  nativeExclusivelyRFQT,
-              }
-            : undefined;
+    const rfqt: Pick<RfqtRequestOpts, 'intentOnFilling' | 'isIndicative' | 'nativeExclusivelyRFQT'> = (() => {
+        if (apiKey) {
+            if (takerAddress) {
```

Comment:
could you make the `if` statement the following? (just a suggestion)

```
if (endpoint === 'quote' && takerAddress) {
                return {
                    intentOnFilling: req.query.intentOnFilling === 'true',
                    isIndicative: false,
                    nativeExclusivelyRFQT,
                };
} else if (endpoint === 'price') {
                return {
                    intentOnFilling: false,
                    isIndicative: true,
                    nativeExclusivelyRFQT,
                };
} else {
    return undefined;
}
```

Reply:
Great suggestion, thank you.  Addressed in https://github.com/0xProject/0x-api/pull/320/commits/e63ee3b02ede33bd30f566ae8db6825d296523d9